clusters/app.ci/prow-monitoring/mixins/_prometheus/dptp_alerts.libsonnet

```diff
@@ -160,6 +160,24 @@
           }
         ],
       },
+      {
+        name: 'openshift-priv-image-building-jobs-failing',
+        rules: [
+          {
+            alert: 'openshift-priv-image-building-jobs-failing',
+            expr: |||
+             sum(rate(prowjob_state_transitions{job="prow-controller-manager",job_name=~".*-images",org="openshift-priv",state="failure"}[6h])) > 0.05
```

Comment:
I am no PromQL expert.

This is how we did it before in a similar occasion (success rate of certain jobs):

https://github.com/openshift/release/blob/6f470d1aa1612de0f7eaec31818feb90694907f5/clusters/app.ci/prow-monitoring/mixins/_grafana_dashboards/build_cop.jsonnet#L128

We could compare the resulting metrics tho.

Reply:
I'll admit to having very little knowledge of libsonnet/jsonnet, so I'm not exactly sure what you are suggesting here. 
I based this loosely off of: https://github.com/openshift/release/blob/3523e00ed9c34bf95a594fae9bb2ab18bcb3e73e/clusters/app.ci/prow-monitoring/mixins/_prometheus/dptp_alerts.libsonnet#L25-L42.

Are you saying I should add it to the `build-cop.jsonnet` file instead?

Reply:
I think that the query in question needs to be reconstructed in a way to calculate all the `-images` prowjobs for the priv org and then get the % of the failed ones. This query calculates only the failed ones so it is impossible to get the % of the failure rate.

Reply:
That makes sense...when I originally wrote the query there was only 1 failing job, so the percentage didn't look that off, but now with more of them it does seem wrong.

I think the correct query is
```
sum(rate(prowjob_state_transitions{job="prow-controller-manager",job_name=~".*-images",org="openshift-priv",state="failure"}[6h])) / sum(rate(prowjob_state_transitions{job="prow-controller-manager",job_name=~".*-images",org="openshift-priv"}[6h])) > 0.05
```

Reply:
Note that this is currently at `13%`, which seems fairly accurate from looking at https://deck-internal-ci.apps.ci.l2s4.p1.openshiftapps.com/?job=*images*

Reply:
> You could be advised with the current deck data. Your query now reports `0.03708952842443732%` failure rate which is not true.

It would be great to understand how Deck gets the numbers which are only available on the deck UI, not metrics and then use them in the alert.

Reply:
I _think_ deck just aggregates the jobs over the last 24 hours (as those are the only ones shown on the page)